### DISCUSSION TITLE
Align margin notes with document title

### DISCUSSION THREAD

**Discussion by mdahirel** [quarto-dev/quarto-cli#6642] [Q&A] [answered]
### Description

Hi,

I would like to set margin notes so that they start at the same height as the title, rather than the main text, and I'm wondering if there's a built-in way to do that that I missed in the documentation. 

(minimal example below, the actual use will be for a CV where the margin notes are used to highlight some skills, and the default way to display them leads to unwanted whitespace)

Cheers

```
---
title: this is an example
format:  
  pdf:
    geometry:
      - a4paper
      - showframe
---

::: {.column-margin}
some stuff to put in the margin
:::

here is some text to put in the main body:

- element 1

- element 2
```
![Capture d’écran 2023-08-29 150624](https://github.com/quarto-dev/quarto-cli/assets/29913635/922b64f6-cc15-4df4-8f68-f17704d22d4d)

**Comment by cscheid** [comment, marked as answer]
I don't think there's an easy way to do that right now, but you can try to hack it the old-fashioned latex way, by adding a rawblock like `` `\vspace{-3em}`{=latex} `` (you will likely need to tweak the amount of spacing)

**Reply by mdahirel** [reply]
Yeah, I expected so. Thanks for the suggestion, works perfectly!

**Comment by baptiste** [comment]
You might want to try the [flowfram](https://ctan.org/pkg/flowfram?lang=en) package, it lets you define frames at specific locations. I used it for [posters](https://b.bapt.xyz/posts/poster/) with quarto ([minimal example here](https://github.com/baptiste/quarto-flowfram/tree/main#readme)).

**Reply by mdahirel** [reply]
Thanks for the suggestion, this looks good! I will have a look at the possibilities later, for now I went with the simpler suggestion by @cscheid
